src/client/dfuse/dfuse.h

```diff
@@ -80,6 +80,8 @@ struct dfuse_projection_info {
 	ATOMIC uint64_t			dfpi_ino_next;
 };
 
+#define READDIR_BLOCKS 8
```

Comment:
It is a bit difficult to understand from the code what this define is used for. Some comments describing from a high level how the different buffers are managed would be helpful.